api/include/opentelemetry/logs/logger.h

```diff
@@ -189,6 +189,23 @@ class Logger
     this->Log(severity, name, "", {}, attributes, {}, {}, {}, std::chrono::system_clock::now());
   }
 
+  /**
+   * Writes a log.
+   * @param severity The severity of the log
+   * @param name The name of the log
+   * @param attributes the attributes, stored as a 2D list of key/value pairs, that are associated
+   * with the log event
+   */
+  void Log(Severity severity,
```

Comment:
`inline`?

Reply:
member function inside class body is `inline` by default. I now noticed that couple of existing member functions have explicit `inline` specifier. Will remove it from there to avoid any confusion.